trocApp/src/entities/Services.ts

```diff
@@ -0,0 +1,18 @@
+interface Services {
+    id: string;
+
+    attributes: {
+        title: string;
+        description: string;
+        localization: string;
+        creationDate: string;
+        state: string;
+        exchangeType: string;
```

Comment:
exchangetype: 'coin' | 'both' | 'mutual' -> Fais une enum au dessus de type ExchangeTypeEnum

Reply:
Pareil pour state waiting, in-progress, finished, canceled